controllers/contentlibrary/clustercontentlibraryitem/clustercontentlibraryitem_controller_intg_test.go

```diff
@@ -94,27 +87,6 @@ func cclItemReconcile() {
 			Expect(ctx.Client.Status().Update(ctx, cclItem)).To(Succeed())
 		})
 
-		AfterEach(func() {
-			Expect(ctx.Client.DeleteAllOf(ctx, &imgregv1a1.ClusterContentLibraryItem{})).Should(Succeed())
-			Eventually(func() bool {
-				cclItemList := &imgregv1a1.ClusterContentLibraryItemList{}
-				if err := ctx.Client.List(ctx, cclItemList); err != nil {
-					return false
-				}
-				return len(cclItemList.Items) == 0
-			}).Should(BeTrue())
-
-			// Manually delete CVMI as envTest doesn't delete it even if the OwnerReference is set up.
```

Comment:
So we don't cleanup CVMI after the test?

Reply:
It's not needed anymore because we are deploying a new item with a unique name in each spec.